mongo_connector/doc_managers/mongo_doc_manager.py

```diff
@@ -114,9 +113,6 @@ def _get_meta_collection(self, namespace):
     def _namespaces(self):
         """Provides the list of namespaces being replicated to MongoDB
         """
-        if self.namespace_set:
-            return self.namespace_set
-
         user_namespaces = []
```

Comment:
It seems as though we use `_namespaces` only to find the collections on the "meta database" that we've created so far, for use in `get_last_doc` and `search`. This whole method could probably be simplified to return only the collection names on `self.meta_database`, instead of probing all databases & collections.